score-client/src/main/java/bio/overture/score/client/config/ClientConfig.java

```diff
@@ -84,11 +82,6 @@ public String clientVersion() {
         ClientConfig.class.getPackage().getImplementationVersion(), "[unknown version]");
   }
 
-  @Bean
-  public CsvParser<KFFileBean> kfFileBeanCsvParser() {
-    return new CsvParser<>(KFFileBean.class, TAB_SEP);
-  }
-
```

Comment:
Due to this deletion we left some unused code. Might be good idea to delete as well?
Take a look if it's safe to remove first for example: 
- `TAB_SEP` class variable, 
- `CsvParser` class
- `jackson-dataformat-csv`  maven dependency.

Reply:
Good suggestion, have updated the PR with following changes